### DISCUSSION TITLE
Consider using infinity as the rate of exponential distribution in deterministic trials

### DISCUSSION THREAD

**Discussion by glopesdev** [SainsburyWellcomeCentre/aeon_experiments#226] [General]
@jkbhagatio I noticed just now that using `Infinity` as the rate of exponential distribution works really nicely across the entire acquisition stack, i.e. sampled values are always 0, the rate itself is displayed as blank on visualizers (rather than a very large spike), and finally in logs the string representation is `Infinity`.

This is preferred since the current threshold will then always stay at exactly the same value, rather than increasing by an infinitesimal.

By the way, you can set any double value to infinity directly in the property grid by pasting the infinity unicode character: ∞

**Comment by jkbhagatio** [comment]
Sounds good, closing this.
